src/storageClients/CassandraTableStorageManager.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
-
-const Promise = require('bluebird');
-
 const BATCH_LIMIT = 10;
 
 module.exports = {
 
-  prepareInsertTopic: topic => { //TODO: the insert statement to reflect the final topic schema
+  prepareInsertTopic: topic => {
     return {
-      query: 'INSERT INTO topics (id, topic, value) VALUES (?, ?, ?)',
-      params: [ topic.id, topic.topic, topic.value ]
+      query: `INSERT INTO computedtopics (
```

Comment:
spark should be the only process writing to this table. I think you meant to write to `watchlist`